selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/CheckConvertingToMavenProjectTest.java

```diff
@@ -0,0 +1,130 @@
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.miscellaneous;
+
+import static java.lang.String.format;
+import static org.eclipse.che.selenium.core.constant.TestMenuCommandsConstants.Project.PROJECT;
+import static org.eclipse.che.selenium.core.constant.TestMenuCommandsConstants.Project.UPDATE_PROJECT_CONFIGURATION;
+import static org.eclipse.che.selenium.core.constant.TestProjectExplorerContextMenuConstants.ContextMenuFirstLevelItems.CONVERT_TO_PROJECT;
+import static org.eclipse.che.selenium.core.project.ProjectTemplates.UNDEFINED_PROJECT;
+
+import com.google.inject.Inject;
+import java.net.URL;
+import java.nio.file.Paths;
+import org.eclipse.che.commons.lang.NameGenerator;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.eclipse.che.selenium.core.action.ActionsFactory;
+import org.eclipse.che.selenium.core.client.TestProjectServiceClient;
+import org.eclipse.che.selenium.core.workspace.TestWorkspace;
+import org.eclipse.che.selenium.pageobject.AskForValueDialog;
+import org.eclipse.che.selenium.pageobject.CodenvyEditor;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.InformationDialog;
+import org.eclipse.che.selenium.pageobject.Loader;
+import org.eclipse.che.selenium.pageobject.Menu;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.Wizard;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/** @author Aleksandr Shmaraev, Musienko Maxim */
+public class CheckConvertingToMavenProjectTest {
+  private static final String       PROJECT_NAME       = NameGenerator.generate("project", 4);
+  private static final String       WEB_APP_MODULE     = "my-webapp";
+  private static final String       NONE_MAVEN_PROJECT = NameGenerator.generate("noneMavenProject", 4);
+  private static final String       PARENT_INFORMATION =
+      "<parent>\n"
+          + "<groupId>org.eclipse.che.examples</groupId>\n"
+          + "<artifactId>qa-multimodule</artifactId>\n"
+          + "    <version>1.0-SNAPSHOT</version>\n"
+          + "</parent>\n";
+  private  static  final String     CONVERT_PATH       = format("%s/%s", PROJECT_NAME, WEB_APP_MODULE);
+  @Inject private TestWorkspace     workspace;
+  @Inject private Ide               ide;
+  @Inject private ProjectExplorer   projectExplorer;
+  @Inject private CodenvyEditor     editor;
+  @Inject private Menu              menu;
+  @Inject private Loader            loader;
+  @Inject private Wizard            wizard;
+  @Inject private AskForValueDialog askForValueDialog;
+  @Inject private InformationDialog informationDialog;
+  @Inject private ActionsFactory           actionsFactory;
+  @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
+  private String workspaceId;
+  @BeforeClass
```

Comment:
Please, add empty line to improve readability.

Reply:
Done